public/index.html

```diff
@@ -29,6 +29,8 @@
 	  },
 	  showStats: true
 	})
+  
+  globalThis.viewer = viewer;
```

Comment:
Is this line from a debugging session? Or does `globalThis.viewer` have a special significance in the implementation?